frontend/src/LanguageSwitcher.js

```diff
@@ -4,9 +4,10 @@ import styled from '@emotion/styled'
 import PropTypes from 'prop-types'
 
 const ButtonLink = styled('button')`
+  font-family: ${({ theme }) => theme.fontSans};
+  font-size: ${({ theme }) => theme.fontSizes[2]};
```

Comment:
Being a bit nitpicky here - would it be better to pass in the fontSizes prop and then set it on the ButtonLink component instead of just passing fontSizes[2]? Would make it easier to change font sizes later (and I'm thinking we may just want to extract ButtonLink later)

Reply:
yeah, probably. I was thinking that we'd want to make a proper styled-systems component at some point, but not before tomorrow. I'll pull out the font size prop as you suggest now though.